src/utils/serializers/arrays.ts

```diff
@@ -1,7 +1,37 @@
+/* eslint-disable no-case-declarations */
+
+const MAX_STRING_CHARS = 100;
```

Comment:
I have limited the number of chars of a string for an array of strings we could support not to cause v8 out of memory. The problem with arrays as you pointed out below is that they are of unknown size so 2 options were available: 
- to use the max number which is hardcoded btw. , but if the array has several elements this would cause v8 out of memory exception
- to put an upper bound as I did (we could increase it for sure), but keeping it below the max 4789655 (the max value).